examples/tutorials/WebsocketServerWorker.ipynb

```diff
@@ -0,0 +1,254 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lrcghOJOWGHw"
+   },
+   "source": [
+    "# WebsocketServerWorker Tutorial\n",
+    "\n",
+    "This tutorial is a 2 notebook tutorial. The partner notebook is the notebook entitled SocketWorker Client.ipynb and is in the same folder as this notebook. You should execute this notebook BEFORE the other.\n",
```

Comment:
Could you explain me the purpose of this notebook? It only starts a WebsocketServerWorker.
In the text it says this is a 2 notebook tutorial. But in this pull request there is only one. Is there a file missing in git?

Additionally, there is a naming convention of the notebook files in this folder. Could you please copy the style from the other notebooks available.

Concerning the setup. No need to install everything again, when using the notebook server started as provided in the pysyft-repo.

Reply:
Actually I only modified a tiny bit of the `Server` notebook downloaded from the Openmined.org website (the same one given by Trask in the Issue #1905 to make it work. So much of this version retains what he did previously. I do agree that it looks kind of rudimentary in its nature and can expand of it if requested. It is to be used on Google Colab hence the weird setup code block... But I need more feedback as to what new things to be included in the new Colab version to continue work.

Reply:
The intension is for the Openmined.org website maintainer to use this version instead of the old non-functional one.

Reply:
There is indeed a companion `Client` notebook. I have opened another issue #2396 and am actively following up on it.